geode-assembly/src/test/java/org/apache/geode/management/internal/cli/commands/StartLocatorCommandTest.java

```diff
@@ -144,4 +159,55 @@ private void addJvmOptionsForOutOfMemoryErrors(final List<String> commandLine) {
       commandLine.add("-XXexitOnOutOfMemory");
     }
   }
+
+  private static final String FAKE_HOSTNAME = "someFakeHostname";
+
+  @Rule
+  public GfshParserRule commandRule = new GfshParserRule();
+
+  @Rule
+  public ExpectedException thrown = ExpectedException.none();
+
+  @Rule
+  public TemporaryFolder temporaryFolder = new TemporaryFolder();
+
+  private StartLocatorCommand spy;
+
+  @Before
+  public void before() throws Exception {
+    spy = Mockito.spy(StartLocatorCommand.class);
+    doReturn(mock(Gfsh.class)).when(spy).getGfsh();
+  }
+
```

Comment:
I'd prefer to see these new @Rule & @Before methods moved to the top of the file with the original @Before method.